src/Acme.UI/Infrastructure/Handlers/PublishHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Umbraco.Core.Composing;
+using Umbraco.Core.Events;
+using Umbraco.Core.Logging;
+using Umbraco.Core.Services;
+using Umbraco.Core.Services.Implement;
+
+namespace Acme.UI.Infrastructure.Handlers
+{
+    [RuntimeLevel(MinLevel = Umbraco.Core.RuntimeLevel.Run)]
+    public class PublishHandlerComposer : ComponentComposer<PublishHandler>
+    {
+    }
+
+    public class PublishHandler : IComponent
+    {
+        private ILogger _logger;
+
+        public PublishHandler(ILogger logger)
+        {
+            _logger = logger;
+        }
+
+        public void Initialize()
+        {
+            // ContentService.Published += this.OnPublish;
+        }
+
+        public void Terminate()
+        {
+            _logger.Info<PublishHandler>("Terminating...");
+        }
+
+        private void OnPublish(IContentService sender, ContentPublishedEventArgs e)
+        {
+            _logger.Info<PublishHandler>("on publish started");
+            foreach (var publishedItem in e.PublishedEntities)
+            {
+                _logger.Info<PublishHandler>(publishedItem.Name + " was published");
```

Comment:
Is this handler doing anything other than just logging?

Reply:
Logging is just place holder. I initially added it for reindexing which is next inline unless we find a way to do it via configuration.